.github/workflows/e2e_lighthouse_performance_acceptance_tests.yml

```diff
@@ -312,8 +312,9 @@ jobs:
           suite:
             - blog-admin-tests/assign-roles-to-users-and-change-tag-properties
             - blog-editor-tests/try-to-publish-a-duplicate-blog-post-and-get-blocked
-            - exploration-editor-tests/load-complete-and-restart-exploration-preview
             - exploration-editor-tests/create-exploration-and-change-basic-settings
+            - exploration-editor-tests/load-complete-and-restart-exploration-preview
+            - exploration-editor-tests/savedraft-publish-and-discard-the-changes
```

Comment:
Reminder to update filename here as well.

Reply:
done